nengo/neurons.py

```diff
@@ -260,6 +311,14 @@ def gain_bias(self, max_rates, intercepts):
         bias = 1 - gain * intercepts
         return gain, bias
 
+    def max_rates_intercepts(self, gain, bias):
+        """Compute the inverse of gain_bias."""
+        intercepts = (1 - bias) / gain
+        max_rates = 1.0 / (self.tau_ref - self.tau_rc * np.log1p(
+            1.0 / (gain * (intercepts - 1) - 1)))
+        max_rates = np.nan_to_num(max_rates)
```

Comment:
Under what conditions do the `max_rates` contain non-finite values? And if they do, wouldn't it be better to keep them that way and have the caller deal with it if they want to convert them to finite values?